pquery-run.sh

```diff
@@ -460,74 +501,46 @@ if [[ $PXC -eq 1 ]]; then
   echo "wsrep-debug=1" >> ${BASEDIR}/my.cnf
   echo "innodb_file_per_table" >> ${BASEDIR}/my.cnf
   echo "innodb_autoinc_lock_mode=2" >> ${BASEDIR}/my.cnf
-  if ! check_for_version $MYSQL_VERSION "8.0.0"; then
-    # TODO: research why is this here?
-    # MariaDB: [ERROR] /test/MD300420-mariadb-10.5.3-linux-x86_64-opt/bin/mysqld: unknown variable 'innodb_locks_unsafe_for_binlog=1'
-    # echo "innodb_locks_unsafe_for_binlog=1" >> ${BASEDIR}/my.cnf
-    # echo "wsrep_sst_auth=$SUSER:$SPASS" >> ${BASEDIR}/my.cnf
-    echo "TODO"
-  else
-    echo "log-error-verbosity=1" >> ${BASEDIR}/my.cnf
-  fi
   echo "wsrep-provider=${BASEDIR}/lib/libgalera_smm.so" >> ${BASEDIR}/my.cnf
   echo "wsrep_sst_method=rsync" >> ${BASEDIR}/my.cnf
-  #echo "wsrep_sst_method=xtrabackup-v2" >> ${BASEDIR}/my.cnf
   echo "core-file" >> ${BASEDIR}/my.cnf
   echo "log-output=none" >> ${BASEDIR}/my.cnf
   echo "wsrep_slave_threads=2" >> ${BASEDIR}/my.cnf
+  echo "wsrep_on=1" >> ${BASEDIR}/my.cnf
   if [[ "$ENCRYPTION_RUN" == 1 ]]; then
     echo "log_bin=binlog" >> ${BASEDIR}/my.cnf
     echo "binlog_format=ROW" >> ${BASEDIR}/my.cnf
     echo "gtid_mode=ON" >> ${BASEDIR}/my.cnf
     echo "enforce_gtid_consistency=ON" >> ${BASEDIR}/my.cnf
     echo "master_verify_checksum=on" >> ${BASEDIR}/my.cnf
     echo "binlog_checksum=CRC32" >> ${BASEDIR}/my.cnf
-    echo "binlog_encryption=ON" >> ${BASEDIR}/my.cnf
-    echo "innodb_sys_tablespace_encrypt=ON" >> ${BASEDIR}/my.cnf
-    echo "pxc_encrypt_cluster_traffic=ON" >> ${BASEDIR}/my.cnf
```

Comment:
Maybe we should make this variable? Maybe we can use that IF statement here (ref below)? What is the alternative?

Reply:
And similar for the other two encryption options thinking about it (i.e. we can use the encryption if for this block of 3 lines), with an else for non-encrypted traffic?

Reply:
some of the encryption options are not valid for MDG. Will add minimal encryption options for encryption run in IF condition.